modules/foreman_compute_profile.py

```diff
@@ -22,7 +22,13 @@
 module: foreman_compute_profile
 short_description: Manage Foreman Compute Profiles using Foreman API
 description:
-- Create and delete Foreman Compute Profiles using Foreman API
+  - Create and delete Foreman Compute Profiles using Foreman API
+version_added: "2.0"
```

Comment:
I think, we should remove that until it is upstream.

Reply:
if everybody is ok, I would open an issue to  be solved before going actually upstream, that includes fixing the number to the actual upstream version.